cats/Teawie-Prison/README.md

```diff
@@ -0,0 +1,19 @@
+<!--
+SPDX-FileCopyrightText: 2024 Sefa Eyeoglu <contact@scrumplex.net>
+
+SPDX-License-Identifier: CC0-1.0
+-->
+
+Teawie Prison
+---
+Teawie Pack, but for some reason he was imprisoned
+
+## Preview
+![Preview](preview.png)
+
+## License
+```
+SPDX-FileCopyrightText: 2024 Sefa Eyeoglu <contact@scrumplex.net>
+
+SPDX-License-Identifier: MIT
```

Comment:
If I understand correctly, the json is MIT while the actual art is cc by-sa. Shouldn't it say both?

Reply:
I guess we would ideally add all licenses and copyright holders, I agree. Lemme update the README template too then

Reply:
done